packages/common/src/sites/themes.ts

```diff
@@ -0,0 +1,54 @@
+import { IPortal } from "@esri/arcgis-rest-portal";
+import {
+  cloneObject,
+  getProp,
+  removeEmptyProps,
+  extend,
+  IHubSiteTheme,
+} from "..";
+
+/**
+ * Default Site Theme
+ */
+export const DEFAULT_THEME: IHubSiteTheme = {
+  header: {
+    background: "#fff",
+    text: "#4c4c4c",
+  },
+  body: {
+    background: "#fff",
+    text: "#4c4c4c",
+    link: "#0079c1",
+  },
+  button: {
+    background: "#0079c1",
+    text: "#fff",
+  },
+  logo: {
+    small: "",
+  },
+  fonts: {
+    base: {
+      url: "",
+      family: "Avenir Next",
+    },
+    heading: {
+      url: "",
+      family: "Avenir Next",
+    },
+  },
+};
+
+/**
+ * Return the default theme, extended with values from the Org's shared theme
+ * @param {Object} portalSelf Org's Portal object
+ */
+export function getOrgDefaultTheme(portalSelf: IPortal) {
```

Comment:
why export this fn _and_ `DEFAULT_THEME`? If it's just for testing, then let's make sure that we don't export it from `./index.ts`

Reply:
DEFAULT_THEME is exported so it can be re-exported from hub-sites package